### PR TITLE
Add secrets capability

### PR DESCRIPTION
**Pull request by oddomatik**
This is a feature we've incorporated for our use internally. Allows mount of secrets into podman containers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,17 @@ podman_services:
...
     # the containers hostname, default is none.
     hostname: chasinglogic.io
+    # Optional: A list of secrets to mount. Secrets will not be created
+    # automatically. The only required field if used is name. The other
+    # values have defaults as specified in podman documentation
+    secrets:
+      - name: secret1
+        type: [mount (default)|env]
+        target: [default: /run/secrets/<secretname> OR env var "<secretname>" if type=env]
+        uid: [0]
```
Comment (chasinglogic) [inline README.md]: can we just set these to the defaults?

File: README.md [hunk 2]
```diff
@@ -64,6 +64,17 @@ podman_services:
     # Optional: Define a hostname for podman's hostname flag. Set's
     # the containers hostname, default is none.
     hostname: chasinglogic.io
+    # Optional: A list of secrets to mount. Secrets will not be created
+    # automatically. The only required field if used is name. The other
+    # values have defaults as specified in podman documentation
+    secrets:
+      - name: secret1
+        type: [mount (default)|env]
```
Comment (chasinglogic) [inline README.md]: I'd prefer if we used comments for documentation and that the values are valid values in the example as much as possible.

File: README.md [hunk 3]
```diff
@@ -64,6 +64,17 @@ podman_services:
     # Optional: Define a hostname for podman's hostname flag. Set's
     # the containers hostname, default is none.
     hostname: chasinglogic.io
+    # Optional: A list of secrets to mount. Secrets will not be created
+    # automatically. The only required field if used is name. The other
+    # values have defaults as specified in podman documentation
```
Comment (chasinglogic) [inline README.md:69]: just slap this link on the end of the comment please: https://docs.podman.io/en/v4.6.0/markdown/options/secret.html

**Review by chasinglogic — CHANGES_REQUESTED** [review]
Thanks for the contribution, just some minor stuff to make the example more copy pastable then I will get this merged.

File: README.md (hunk 2 above)
Comment (oddomatik) [reply README.md]: I believe I've understood but feel free to elaborate!

**Comment by chasinglogic** [comment]
Perfect, thanks again!

**Review by chasinglogic — APPROVED** [review]

**Comment by chasinglogic** [comment]
This has now been released in version 1.3.0

**Pull request merged**